config/locales/en.yml

```diff
@@ -129,10 +131,29 @@ en:
               noopener">Learn more about Vocdoni</a>.
             title: Elections (Vocdoni)
           new:
-            create: Create
+            basic_info: Basic info
+            census: Census
+            next_button: Done, go to the next step
+            publish: Publish
+            publish_text: "If you are happy with your configuration (you can still
+              edit all the steps), now it's time to publish and setup de election.<br><br>
+              \nTo setup the election you must publish it first.<br><br> \nThe setup
+              process is when the election configuration, including the census, will
+              be sent to the Vocdoni platform. This means it will be unmodifiable
+              after that, due the use of the blockchain. After the setup, the election\ncan
+              start (either manually or automatically depending on the configuration).\n"
```

Comment:
I have a couple suggestions here:
* If the string has html, we should add the suffix "_html" so we don't need to call to the html_safe method on the view
* Also I think it's cleaner and more semantic to use the paragraph tag (`<p>`) instead of the break line (`<br>`)
* Finally, if rubocop let us, I prefer to break the 80 characters column convention if we improve the legibility.

This is what I have in mind:  
```suggestion
            publish_text_html: |-
              <p>If you are happy with your configuration (you can still edit all the steps), now it's time to publish and setup de election.</p>
               <p>To setup the election you must publish it first.</p>
               <p>The setup process is when the election configuration, including the census, will be sent to the Vocdoni platform. This means it will be unmodifiable
                after that, due the use of the blockchain. After the setup, the election can start (either manually or automatically depending on the configuration).</p>

```

We would need to change it also on `publish_text_all_done` and also on the view (the `_html` suffix in the variables on both of these and removing the `html_safe` method call).